core-blocks/gallery/edit.js

```diff
@@ -195,6 +198,13 @@ export default class GalleryEdit extends Component {
 			</BlockControls>
 		);
 
+		const noticesUI = notices.noticeList.length > 0 &&
+			<BlockNotices
```

Comment:
I'm just wondering here... Could the `BlockNotices` be a prop of this component that is added by `withNotices`. Or would there be any other way to avoid passing all this stuff (less for the block author to worry about).